sandboxnu/office-hours
# Question draft in progress indicator

<img width="1157" alt="Screen Shot 2020-07-20 at 11 02 35 AM" src="https://user-images.githubusercontent.com/33535106/87953055-8d13e900-ca78-11ea-8042-152ae52b4813.png">

Testing steps:
- Go to the queue Page
- You should not have a draft in local storage, so do not expect the popup. 
- Start to edit your question, do not finish question, reload page
- Note the popup, click continue drafting question. See your question from before. Then do not finish again and reload the page
- Delete question, refresh. (no popup)
